html-web-page/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <!-- your HTML code here 👇 -->
+  <header>
+    <div>
+      <h1>Jelly of the Month Club</h1>
+      <h2>Make all your friends jelly of your stock of jelly!</h2>
+    </div>
+  </header>
+
+  <div>
+    <p>
+      Here at JOTM Inc., we are constantly striving to supply our customers
+      with the highest-quality and most unique jellies on a monthly basis.
+      For the jelly lover in your life (or yourself!),
+      this is the gift that keeps on giving all year round.
+      We offer three tiers of monthly jelly joy,
+      and each one offers their own perks.
+      Here is a breakdown of each, as well as your monthly price:
+    </p>
+  </div>
+
+  <table>
+    <thead>
+      <tr>
+        <th scope="col"> </th>
+        <th scope="col">Basic Plan</th>
+        <th scope="col">Deluxe Plan</th>
+        <th scope="col">Jelly Giant Plan</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Three new curated jellies per month</td>
+        <td>&check;</td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Monthly jellies come with commemorative mason jars</td>
+        <td> </td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free bonus jar of preserves bi-monthly</td>
+        <td> </td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free yearly tickets to Jelly Con</td>
+        <td> </td>
+        <td> </td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Monthly pricing</td>
+        <td>$30</td>
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <div>
+    <p> </p>
```

Comment:
only change I would recommend is to remove this element. You don't want to have any empty tags in the HTML